test/testEqWrappedPrototype.tex

```diff
@@ -42,7 +48,8 @@
 k=\Theta\left(\Theta\left( t \right)-0.5\right)\Theta\left(\Theta\left( t \right)+\delta\left( t \right)-0.5\right)+0+0
 \end{dmath*}
 \begin{dmath*}
-l=\left(1-\Theta(0.5-max\left(\Theta\left(\Theta\left(\mathrm{Repay}- t \right)+\delta\left( t -\mathrm{Repay}\right)-0.5\right),\Theta\left(\Theta\left( t \right)\right)\right)\right)
ight)+\Theta\left(\Theta\left( t \right)-0.5\right)+\Theta\left(\Theta\left( t \right)-0.5\right)
```

Comment:
Whoopsies! Thanks for catching and fixing that.

Reply:
Was not aware of them. TexMaker sorted them out. :)